User_Story#1/SortingApp/BusinessLayer/Utilities/Validator/СloudFileValidator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using BusinessLayer.Utilities.Validator.Interfaces;
+using System.Net;
+
+namespace BusinessLayer.Validator
+{
+    /// <summary>
+    /// Checks whether content of cloud doc is valid.
+    /// </summary>
+    /// <owner>Anton Petrenko</owner>
+    /// <seealso cref="BusinessLayer.Interfaces.IValidator" />
+    public class СloudFileValidator : IValidator
+    {
+        /// <summary>
+        /// Determines whether data exists.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        /// <param name="content">The content of cloud doc.</param>
+        /// <returns>
+        ///   <c>true</c> if data exists; otherwise, <c>false</c>.
+        /// </returns>
+        public bool IsDataExist(string url)
+        {
+            if (url is null)
```

Comment:
Url could contain empty string or spaces.
In such cases we should call method
string.IsNullOrWhiteSpace()